src/index.ts

```diff
@@ -1 +1,5 @@
-export { atomWithMachine, RESTART } from './atomWithMachine.js';
+export { atomWithMachine } from './atomWithMachine.js';
+export { isGetter, RESTART } from './utils.js';
+export type { Gettable } from './utils.js';
```

Comment:
Can we avoid exporting `isGetter` and `Gettable`. They feel internal things.
```suggestion
export { RESTART } from './utils.js';
```